test/controllers/callback_controller_test.rb

```diff
@@ -333,6 +352,32 @@ class CallbackControllerTest < ActionController::TestCase
       assert_response 302
     end
 
+    test "#callback calls post_authenticate_tasks if custom_post_authenticate_tasks is set" do
```

Comment:
Should we have a very simple test here to ensure that the default behaviour gets called (even if we have to stub it)?